scripts/src/main/resources/scripts/command/eclipse

```diff
@@ -164,6 +164,68 @@ function doStartEclipse() {
   fi
 }
 
+function doStartEclipseBlocking() {
+  if doIsWindows
+  then
+    # shellcheck disable=SC2086
+    "eclipse" /B "${ECLIPSE_HOME}/eclipsec" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/javaw" -showlocation "${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  else
+    # shellcheck disable=SC2086
+    "${ECLIPSE_HOME}/eclipse" -clean -data "${WORKSPACE_PATH}" -keyring ~/.eclipse/.keyring -vm "${JAVA_HOME}/bin/java" -showlocation "${DEVON_IDE_HOME/*\//}/${WORKSPACE}" -vmargs ${ECLIPSE_VMARGS}
+  fi
+}
+
+# $1: project type
+# $2: optional project path
+function doImportProject() {
+  local setup_ws="${WORKSPACE_PATH}/.metadata/.plugins/org.eclipse.oomph.setup/workspace.setup"
+  local setup_templ="${DEVON_IDE_HOME}/scripts/templates/oomph"
+  local root_path="${OLDPWD}"
+  if [ "${1}" = "mvn" ]
+  then
+    setup_templ="${setup_templ}/import_mvn.setup"
+  elif [ "${1}" = "plain" ]
+  then
+    setup_templ="${setup_templ}/import_plain.setup"
+  else
+    echo "You must specify the project type (mvn/plain)!"
+    return
+  fi
+
+  if [ -n "${2}" ]
+  then
+    root_path="${2}"
+  fi
+  if [ ! -f "${setup_ws}" ]
+  then
+    echo "Could not find file 'workspace.setup' in C:${setup_ws:2:-15}"
+    echo "Please run 'devon eclipse start' in this workspace first ..."
+    return
+  fi
+  if [ -f "${setup_ws}.bak" ]
+  then
+  	echo "A backup of 'workspace.setup' already exists in C:${setup_ws:2:-15}"
+    echo "Skipping backup creation ..."
+  else
+    cp "${setup_ws}" "${setup_ws}.bak"
+  	echo "Created backup of 'workspace.setup' in C:${setup_ws:2:-15}"
+  fi
+  cp -r "${setup_templ}" "${setup_ws}"
+  sed -i "s~@ROOT@~${root_path:2}~" "${setup_ws}"
```

Comment:
Why skips this the first 2 characters? ( due to the offset 2 in `${root_path:2}` ) Is this to remove some leading "/c" from path name? And how will `mvn` then  - running inside the same environment - be able to detect that path. I'd say this should be t
I have not yet checked it myself inside the different environments, but I can't imagine this will work everywhere. Or do I miss something here?

Reply:
So Ooomp just plainly adds the drive letter where the running Eclipse instance is located and — on Windows — converts "`/`" to "`\`". In the following (failed) attempt my `.setup` file included the following `${root_path}`:
> `/c/Users/phschimm/Downloads/devon-ide-scripts-3.0.0-beta22/workspaces/main/jump-the-queue-v2`

![oomph_path_transform](https://user-images.githubusercontent.com/53938950/64956243-71551a80-d88a-11e9-947f-e23adcac0c3f.png)

So I've decided to use `echo ${root_path} | cut -d'/' --complement -f2` to exclude the drive-letter from the path. Since this is based on the delimiter "`/`" and the `--fields` argument, it should work no matter what environment we are in (at least I can't imagine a scenario right now where it wouldn't).

By the way, I'm now working on [this branch](https://github.com/devonfw/ide/tree/eclipse-import).